modules/machineconfig-modify-dns-nameservers.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machineconfig-modify-dns-nameservers_{context}"]
+= Modifying the DNS name servers
+
+You can override the DNS name servers and search list in the `/etc/resolv.conf` file.
+Some operating systems use `NetworkManager` to manage network interfaces.
+When `NetworkManager` manages some interfaces, it also manages the contents of the `/etc/resolv.conf` file.
```

Comment:
It might be worth checking RHEL docs, but I think it goes by just _NetworkManager_ without italics.

Reply:
I'm using backticks based on the guidance for a daemon, service, or software package.